Sources/Workspace/WorkspaceConfiguration.swift

```diff
@@ -66,8 +66,13 @@ extension Workspace {
         public var sharedManifestsCacheDirectory: AbsolutePath? {
             self.sharedCacheDirectory.map { DefaultLocations.manifestsDirectory(at: $0) }
         }
+        
+        /// Path to the shared fingerprints cache.
+        public var sharedFingerprintsCacheDirectory: AbsolutePath? {
```

Comment:
is "cache" a good term here? maybe just sharedFingerprintsDirectory ?

Reply:
I chose cache dir because these are data that can be deleted and recaptured if needed. I am open to use `sharedConfigurationDirectory` instead if we think this is more appropriate.

Reply:
Decided to do `sharedFingerprintsDirectory` at `~/.swiftpm/security/fingerprints` instead